CHANGELOG.md

```diff
@@ -31,7 +36,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     
 ### Fixed
 - Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.
+=======
+- Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.
+- Add a workaround to avoid 480p resolution scale down when there are 5-8 videos for the default video uplink policy 
+  for Chromium browsers version 98 on Windows and use 360p instead.
+>>>>>>> 6065a6ce (Updated change log)
```

Comment:
Merge conflict? Your changes probably need to be under 3.0.0-beta.1

Reply:
fixed it.